zaza/charm_lifecycle/func_test_runner.py

```diff
@@ -28,6 +28,49 @@
 import zaza.utilities.run_report as run_report
 
 
+def run_env_deployments(env_deployment, keep_model=False):
```

Comment:
The plural form of the function name gives the impression it will take a ``List`` of ``EnvironmentDeploy`` objects and deploy the models defined in them, while I think what is actually happening is it takes a single ``EnvironmentDeploy`` object and deploys the models defined in it.